app_flutter/lib/service/google_authentication.dart

```diff
@@ -24,30 +33,30 @@ class GoogleSignInService {
   static const List<String> _googleScopes = <String>[
     'https://www.googleapis.com/auth/userinfo.email',
     'https://www.googleapis.com/auth/userinfo.profile',
+    'openid',
   ];
 
-  // TODO(chillers): Switch to official Flutter plugin when it supports web.
   final GoogleSignIn _googleSignIn;
 
-  AuthCredentials _credentials;
-
-  GoogleAccount _user;
-
   /// Whether or not the application has been signed in to.
-  bool get isAuthenticated => _credentials?.accessToken != null;
+  bool get isAuthenticated => user != null;
 
-  /// The profile photo url of the current user signed in.
-  String get avatarUrl => _user?.photoUrl;
-
-  /// The email of the current user signed in.
-  String get email => _user?.email;
+  /// The Google Account for the signed in user, null if no user is signed in.
+  ///
+  /// Read only object with only access to clear client auth tokens.
+  GoogleSignInAccount user;
 
   /// Authentication token to be sent to Cocoon Backend to verify API calls.
-  String get accessToken => _credentials?.accessToken;
+  Future<String> get idToken => user?.authentication
+      ?.then((GoogleSignInAuthentication key) => key.idToken);
 
   /// Initiate the Google Sign In process.
   Future<void> signIn() async {
-    _credentials = await _googleSignIn.signIn();
-    _user = await _googleSignIn.getCurrentUser();
+    user = await _googleSignIn.signIn();
+  }
+
+  Future<void> signOut() async {
+    await _googleSignIn.signOut();
```

Comment:
No action required, just want to mention that it's amusing to see "SignIn.signOut" :)